meta-avocado/recipes-avocado/packagegroups/packagegroup-avocado-sdk.bb

```diff
@@ -10,16 +10,6 @@ SDK_TOOLCHAIN_DEPENDS = " \
   nativesdk-avocado-pkg-rootfs \
   nativesdk-avocado-pkg-initramfs \
   nativesdk-avocado-pkg-images \
-  nativesdk-qemu \
-  nativesdk-qemu-helper \
-  nativesdk-bmaptool \
-  nativesdk-python3-pip \
-  nativesdk-make \
-  nativesdk-cmake \
-  nativesdk-squashfs-tools \
-  ${@bb.utils.contains('SDK_TOOLCHAIN_LANGS', 'go', 'packagegroup-go-cross-canadian-${MACHINE}', '', d)} \
-  ${@bb.utils.contains('SDK_TOOLCHAIN_LANGS', 'rust', 'packagegroup-rust-cross-canadian-${MACHINE}', '', d)} \
-  ${@bb.utils.contains('DISTRO_FEATURES', 'wayland', 'nativesdk-wayland-tools nativesdk-wayland-dev', '', d)} \
```

Comment:
I can see why Wayland is an SDK extra. But Go and Rust? Most devs will be offended.